tests/test_plugins.py

```diff
@@ -0,0 +1,143 @@
+import tempfile
+import strax
+import straxen
+import numpy as np
+from immutabledict import immutabledict
+from strax.testutils import run_id, recs_per_chunk
+
+# Number of chunks for the dummy raw records we are writing here
+N_CHUNKS = 2
+
+##
+# Tools
+##
+
+@strax.takes_config(
+    strax.Option('secret_time_offset', default=0, track=False)
+)
+class DummyRawRecords(strax.Plugin):
+    """
+    Provide dummy raw records for the mayor raw_record types
+    """
+    provides = ('raw_records', 'raw_records_he', 'raw_records_nv',)
+    parallel = 'process'
+    depends_on = tuple()
+    data_kind = immutabledict(zip(provides, provides))
+    rechunk_on_save = False
+    dtype = {p: strax.raw_record_dtype() for p in provides}
+
+    def source_finished(self):
+        return True
+
+    def is_ready(self, chunk_i):
+        return chunk_i < N_CHUNKS
+
+    def compute(self, chunk_i):
+        r = np.zeros(recs_per_chunk, self.dtype['raw_records'])
+        t0 = chunk_i + self.config['secret_time_offset']
+        r['time'] = t0
+        r['length'] = r['dt'] = 1
+        r['channel'] = np.arange(len(r))
+
+        res = {p: self.chunk(start=t0, end=t0 + 1, data=r, data_type=p)

```

Comment:
I think we can have here easily two birds with one stone. Maybe just add an additional chunk which is empty to test this here as well.

Reply:
Good idea, will do.

Reply:
In 0d09964 I indeed found a bug (which I might have induced) in the Ledplugin using an empty chunk, thanks Daniel!